### PR TITLE
chore: add RFCs blog

### PR DESCRIPTION
**Pull request by erickzhao**
@electron/wg-api

### CODE REVIEW

**Review by dsanders11 — CHANGES_REQUESTED** [review]
The Mermaid diagram isn't rendering on the site. [Looks like we need to configure it.](https://docusaurus.io/docs/2.x/markdown-features/diagrams)

Screenshot 2024-02-12 at 3 04 18 PM

**Comment by erickzhao** [comment]
newest deploy works now

File: package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "@docusaurus/core": "^2.4.3",
     "@docusaurus/plugin-google-analytics": "^2.4.3",
     "@docusaurus/preset-classic": "^2.4.3",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.4.3",
+    "@docusaurus/remark-plugin-npm2yarn": "",
```
Comment (dsanders11) [inline package.json:38]: Was this change intentional?

**Review by ckerr — COMMENTED** [review]
mermaid aside, the text of the blog post LGTM :+1:

File: blog/rfcs.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  P[Proposed] --> PC[Pending Comment];
+  PC --> A[Active];
+  A --> C[Completed];
+  PC --> P;
+  P --> D[Declined];
+  A --> D;
+  style D fill:red;
+  style A fill:orange;
+  style C fill:lightgreen;
+  style PC fill:yellow;
```
Comment (dsanders11) [inline blog/rfcs.md:45]:
```suggestion
  style P color:black;
  style P fill:white;
  style D color:black;
  style D fill:red;
  style A color:black;
  style A fill:orange;
  style C color:black;
  style C fill:lightgreen;
  style PC color:black;
  style PC fill:yellow;
```

The current styling is difficult to read in dark mode due to the way the `@docusaurus/theme-mermaid` package selects the text color such that it doesn't have much contrast with the color fills.

Explicitly set the text color (and set a white fill for `P`) so that it is legible in both light and dark mode.

Before:

Screenshot 2024-02-12 at 11 19 13 PM
Screenshot 2024-02-12 at 11 19 26 PM

After:

Screenshot 2024-02-12 at 11 19 59 PM
Screenshot 2024-02-12 at 11 20 10 PM

File: package.json (hunk 1 above)
Comment (erickzhao) [reply package.json:38]: no, not sure how that snuck into there!

**Review by dsanders11 — APPROVED** [review]
